crates/types/src/simple_vote.rs

```diff
@@ -0,0 +1,214 @@
+#![allow(dead_code)]
+#![allow(clippy::missing_docs_in_private_items)]
+#![allow(missing_docs)]
+
+use std::{clone, fmt::Debug, hash::Hash, marker::PhantomData};
+
+use commit::{Commitment, Committable};
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    traits::{
+        election::Membership,
+        node_implementation::NodeType,
+        signature_key::{EncodedPublicKey, EncodedSignature, SignatureKey},
+    },
+    vote2::{HasViewNumber, Vote2},
+};
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Hash, Eq)]
+pub struct YesData<LEAF: Committable> {
+    pub leaf_commit: Commitment<LEAF>,
+}
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Hash, Eq)]
```

Comment:
I feel we should remove all references to "No" votes, since we are not going to implement them in the near future.

Reply:
Yeah we can remove, I'm going to push very soon, hopefully something that builds, and I will remove the NoVote stuff